query_suggestions/wildcard_more.md

```diff
@@ -0,0 +1,6 @@
+WQL supports only the asterisk (*) character as a wildcard.
+
+**Docs:**
+
+[Optimize dashboard performance](https://docs.wavefront.com/ui_dashboards.html#ensure-optimal-dashboard-performance)<br>
+RK>>Please add this link: [Wavefront Query Language Reference](https://docs.wavefront.com/query_language_reference.html#partial-regex-wildcards-aliases-and-variables)
```

Comment:
Here, for the discussion of wildcards, we want to add this link. But please remove "RK>>Please add this link:"